membership_withdrawal/views/membership_withdrawal_reason_view.xml

```diff
@@ -16,10 +16,22 @@
     <field name="arch" type="xml">
         <tree string="Membership withdrawal reasons" editable="top">
             <field name="name"/>
+            <field name="active" readonly="1" />
         </tree>
     </field>
 </record>
 
+<record id="membership_withdrawal_reason_search_view" model="ir.ui.view">
+    <field name="model">membership.withdrawal_reason</field>
+    <field name="arch" type="xml">
+        <search>
+            <filter string="All" domain="['|', ('active','=', True), ('active','=', False)]" name="all_reasons"/>
+            <filter string="Active" domain="[('active','=', True)]" name="active_reasons"/>
+            <filter string="Archived" domain="[('active','=', False)]" name="archived_reasons"/>
```

Comment:
Only the "Archived" filter is needed, the other two filters are not strictly necessary and could be removed.

Reply:
Although the `All` filter can be removed, the other is useful for allowing to see both together (active and non active). They should be anyway split with `<separator>` elements for not combining with other filter elements.